test/format.spec.js

```diff
@@ -54,3 +55,36 @@ describe('Format', () => {
     });
   }
 });
+
+const ESLINT_PATH = path.resolve('./node_modules/eslint/bin/eslint.js');
+
+function runEslint(cwd) {
+  const result = spawnSync(ESLINT_PATH, ['**/*.js'], { cwd });
+  result.stdout = result.stdout && result.stdout.toString();
+  result.stderr = result.stderr && result.stderr.toString();
+  return result;
+}
+
+const codebases = [
+  'project-1'
+];
+
+describe('Check codebases', () => {
+  for (const folder of codebases) {
+    it(`${folder} - eslint should give expected output`, done => {
+      const fullFolder = path.resolve(`./test/codebases/${folder}`);
+
+      // Spawn a eslint process
+      const { stdout, stderr } = runEslint(fullFolder);
+
+      const regexp = new RegExp(`^${fullFolder.replace(/[\\^$.*+?()[\]{}|]/g, '\\$&')}.+\\.js$`, 'gm'); // Escape regexp
+
+      // Strip root from filenames
+      expect(stdout.replace(regexp, match => match.replace(fullFolder, '.').replace(/\\/g, '/'))).toMatchSnapshot();
+
+      expect(stderr).toEqual('');
+
+      done();
```

Comment:
We use jest, which doesn't require using done for tests. Try removing the `done()` and see if the tests still behave as expected